Added accounts

ananysharma77 opened the pull request.

ananysharma77 committed:
Added accounts

benubah reviewed (commented):
Please make the required changes to your PR to reflect Twitter accounts focused on R

ananysharma77 commented:
I have added eight more accounts and removed the ones which were
irrelevant. The previous PR is closed, a new one has come up.

On Tue, Jun 16, 2020 at 11:42 PM Ben <notifications@github.com> wrote:

> *@benubah* commented on this pull request.
>
> Please make the required changes to your PR to reflect Twitter accounts
> focused on R
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/r-community/tweet-explorer/pull/6#pullrequestreview-431768843>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIHDKB7TQDCSUXX4ZTFWQ7LRW6YXRANCNFSM4N7QFWGA>
> .
>
